kitsune/search/v2/base.py

```diff
@@ -329,8 +329,9 @@ def run(self, query, page=1, default_operator="AND", **kwargs):
             self.get_filter(query=query, default_operator=default_operator, **kwargs)
         )
         # add highlights for the search class' highlight_fields
-        search = search.highlight(*self.get_highlight_fields(), **self.get_highlight_options())
-
+        if highlight_options := self.get_highlight_options():
+            highlight_options = highlight_options[0]
```

Comment:
This'll exhibit quite weird behaviour if we set different highlight options across child Searches, but let's fix it in a separate PR: https://github.com/mozilla/sumo-project/issues/819